packages/gatsby-core-utils/src/create-require-from-path.ts

```diff
@@ -1,22 +1,18 @@
 import Module from "module"
 import path from "path"
 
-// Polyfill Node's `Module.createRequireFromPath` if not present (added in Node v10.12.0)
-const createRequire = (filename: string): NodeRequire => {
-  if (`createRequire` in Module) {
-    return Module.createRequire(filename)
-  }
-  if (`createRequireFromPath` in Module) {
-    return Module.createRequireFromPath(filename)
-  }
-
+const fallback = (filename: string): NodeRequire => {
+  // @ts-ignore
```

Comment:
why is this (and next one) `@ts-ignore` needed? Might be worth adding context? (or if possible not use the ignore directives :P)

Reply:
I agree. This is suppressing a valid warning.